peer/brontide.go

```diff
@@ -987,9 +977,7 @@ func (p *Brontide) addLink(chanPoint *wire.OutPoint,
 				"are not yet taproot channel compatible",
 				chanPoint)
 		}
-	case chanType.HasAnchors():
-		towerClient = p.cfg.AnchorTowerClient
-	default:
+	} else {
```

Comment:
I think we can remove this `else` completely and directly use `p.cfg.TowerClient` everywhere?

Reply:
dont think we want to do that for tap chans at the moment though?

ie we want tap chans to see it as nil for now

Reply:
Ah, yes, makes sense.